lib/listen.js

```diff
@@ -29,7 +29,7 @@ export default function() {
       swap.stateId = stateIds[stateIds.length - 1];
     }
 
-    if (location.hash) {
+    if (location.hash.match(/^#pane=/)) {
```

Comment:
A cleaner approach would be to use `test`. So something like this:

`/^#pane=/.test(location.hash)`

Also, I'm not sure we can count on the pane param always being the first param. We might want to do:

`/pane=/.test(location.hash)`

instead

Reply:
Updated